package.json

```diff
@@ -6,15 +6,14 @@
   "dependencies": {
     "arp-listener": "0.0.0",
     "dominos": "^1.0.5",
-    "dotenv": "^1.2.0",
     "i": "^0.3.3",
     "node-dash-button": "^0.1.1",
     "prompt": "^0.2.14"
   },
   "devDependencies": {},
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "sudo node app.js"
+    "start": "sudo nodejs app.js"
```

Comment:
I don't know if I approve of this. This messes up for other users not using Ubuntu like OS X. This should be pointed out as a possible "issue" in the README to install nodejs legacy. I also will create a PR after this is approved to update and remove a few unnecessary dependencies.